NativeApp/src/components/Booking/container.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import { Alert } from 'react-native';
+import { PropTypes as PT } from 'prop-types';
+import moment from 'moment';
+import { userProfile } from '../../utilities/currentUser';
+import { requestHolidays } from '../../services/holidayService';
+
+
+export default Container => class extends Component {
+  static propTypes = {
+    navigation: PT.shape({
+      navigate: PT.fun,
+    }).isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      booking: {
+        startDate: '',
+        endDate: '',
+        halfDay: false,
+      },
+      user: {},
+    };
+  }
+
+  componentDidMount() {
+    const { navigation } = this.props;
+    const chosenDate = navigation.getParam('date', '');
+
+    userProfile()
```

Comment:
Should this be renamed to `getUserProfile`? I think it would make more sense.

Reply:
I've actually got a service call called `getUserProfile`.  @bgribz can't think of another name for it. Any other options?

Reply:
Oooh ok, em it's not terribly important, the code after makes it clear enough I guess!